### PR TITLE
[Merged by Bors] - set limits for all pods in ci and reenable debug logs

### PR DESCRIPTION
**Pull request by dshulyak**
if limits and requests are both set pod will have guaranteed qos in k8s. 
this should prevent environment issues where some of the nodes go away in the middle of the test

this still doesn't prevent problems with poet, where some of them perform significantly worse than other
but that should not affect tests as we expect to select the best poet proof.

i opened two bugs based on failures observed here:
- https://github.com/spacemeshos/go-spacemesh/issues/3908
- https://github.com/spacemeshos/go-spacemesh/issues/3907
- https://github.com/spacemeshos/go-spacemesh/issues/3909

i think it is also obvious that none of it has to do with debug log level, but i can keep using info if this is preferred

### CODE REVIEW

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
one of the pods (smesher-40-0) decided to skip hare in layer 15 :/

image

updated: https://github.com/spacemeshos/go-spacemesh/issues/3909

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors try

**Review by countvonzero — APPROVED** [review]

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors try

File: systest/testcontext/context.go [hunk 1]
```diff
@@ -314,7 +314,7 @@ func New(t *testing.T, opts ...Opt) *Context {
 		Image:             imageFlag.Get(p),
 		PoetImage:         poetImage.Get(p),
 		NodeSelector:      nodeSelector.Get(p),
-		Log:               zaptest.NewLogger(t, zaptest.Level(logLevel)).Sugar(),
+		Log:               zaptest.NewLogger(t, zaptest.Level(logLevel)).Sugar().With("test", t.Name()),
```
Comment (poszu) [inline systest/testcontext/context.go]: How about naming it instead:
```suggestion
		Log:               zaptest.NewLogger(t, zaptest.Level(logLevel)).Sugar().Named(t.Name()),
```

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors merge

**Comment by countvonzero** [comment]
bors merge

**Pull request closed without merging**